main.go

```diff
@@ -96,7 +97,14 @@ func main() {
 	} else if *dashboard {
 		startDashboardServer(c, *auditPath, *dashboardPort, *dashboardBasePath)
 	} else if *audit {
-		runAudit(c, *auditPath, *setExitCode, *auditOutputFile, *auditOutputURL, *auditOutputFormat)
+		auditData := runAudit(c, *auditPath, *auditOutputFile, *auditOutputURL, *auditOutputFormat)
+		if *setExitCode && auditData.ClusterSummary.Results.Totals.Errors > 0 {
+			logrus.Infof("%d errors found in audit", auditData.ClusterSummary.Results.Totals.Errors)
```

Comment:
I think it might be useful for the end user to know the score at this point too so they can adjust the flags armed with some knowledge if they're playing with what's important to them.

I'm happy to chat about it if you want to consider it.

Reply:
So still reading through all this and looks like `runAudit()` still outputs the report to stdout and the flags just add a note to stderr of the errors found and if the reported score is lower than the desired minimum 

:+1: 

I was confused because auditData := runAudit() has a side-effect of printing to stdout which wasn't immediately clear to me since we're storing "outputs" to a variable.

Reply:
Good point - renamed to `runAndReportAudit`